toolchain/check/context.cpp

```diff
@@ -505,7 +505,7 @@ auto Context::AppendLookupScopesForConstant(
   return false;
 }
 
-auto Context::LookupQualifiedName(SemIRLoc loc, SemIR::NameId name_id,
+auto Context::LookupQualifiedName(SemIR::LocId loc, SemIR::NameId name_id,
```

Comment:
```suggestion
auto Context::LookupQualifiedName(SemIR::LocId loc_id, SemIR::NameId name_id,
```

As above (also, more cases in this PR -- I'm not going to keep commenting)